desktop/core/src/desktop/lib/django_forms.py

```diff
@@ -51,7 +51,7 @@
 try:
   from django.utils.encoding import StrAndUnicode
 except ImportError:
-  from django.utils.encoding import python_2_unicode_compatible
+  from six import python_2_unicode_compatible
```

Comment:
For the record this is a no-op in Python 3
https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.encoding.python_2_unicode_compatible